udata/core/dataservices/models.py

```diff
@@ -140,11 +140,23 @@ def filter_by_topic(base_query, filter_value):
         )
 
 
+def filter_by_reuse(base_query, filter_value):
+    from udata.core.reuse.models import Reuse
+
+    try:
+        reuse = Reuse.objects.get(id=filter_value)
+    except Reuse.DoesNotExist:
+        pass
+    else:
+        return base_query.filter(id__in=[dataservice.id for dataservice in reuse.dataservices])
+
+
 @generate_fields(
     searchable=True,
     nested_filters={"organization_badge": "organization.badges"},
     standalone_filters=[
-        {"key": "topic", "constraints": "objectid", "query": filter_by_topic, "type": str}
+        {"key": "topic", "constraints": "objectid", "query": filter_by_topic, "type": str},
+        {"key": "reuse", "constraints": "objectid", "query": filter_by_reuse, "type": str},
```

Comment:
Seems like constraints [should be a list](https://github.com/opendatateam/udata/blob/master/udata/api_fields.py#L520)?